CMPUT301Project/app/src/main/java/com/example/GroupProject/MainActivity.java

```diff
@@ -54,28 +54,27 @@ protected void onCreate(Bundle savedInstanceState) {
             startActivity(intent);
         }
 
-
-        ProfileFragment profileFragment = new ProfileFragment();
-        HabitFragment habitFragment = new HabitFragment();
-
```

Comment:
Clicking the back button in an activity gets you back to the fragment you entered the activity from